lighthouse/src/main.rs

```diff
@@ -204,7 +204,7 @@ fn main() {
                 .expect("Debug-level must be present")
                 .into();
 
-            boot_node::run(bootnode_matches, eth_spec_id, debug_info);
+            boot_node::run(&matches, bootnode_matches, eth_spec_id, debug_info);
```

Comment:
As per earlier comment, I believe `debug_info` is inside `matches` so we can probably remove it here and parse it inside of `boot_node` as well.

Reply:
While working on the changes I was about to do this, but then realized it moves the possibility of [this](https://github.com/mooori/lighthouse/blob/37c3bf2cc44db2d8fc40b5abe1fbe889482fb128/lighthouse/src/main.rs#L204) unhandled panic from `main` to `boot_node::run`.

If that's okay, I'm happy to move parsing of `debug-level` to `boot_node::run`.

Or we change `boot_node::run` to return a `Result` and then keep the panic in `main`?

Reply:
Ah yes, that's fair. I don't *think* there would be any problem with doing that, but there's probably no need to rock the boat on this one right now.